app/apis/hackney_api/repairs_client.rb

```diff
@@ -191,6 +191,13 @@ def get_property_block_work_orders_by_trade(reference:, trade:, date_from:, date
       []
     end
 
+    def get_cautionary_contact(reference)
```

Comment:
is that a reference to the property or to the contact?

Reply:
that's a property reference

Reply:
We should probably call it `get_cautionary_contact_by_property_reference` then, even though it's long and ugly. Otherwise I would expect to pass it a cautionary contact id.